Management/src/main/java/com/stack/jspdemo/controller/ProductREST.java

```diff
@@ -0,0 +1,24 @@
+package com.stack.jspdemo.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.stack.jspdemo.model.Product;
+import com.stack.jspdemo.service.ProductService;
+
+@RestController
+public class ProductREST {
+
+	@Autowired
+	ProductService productService;
+	@Autowired
```

Comment:
Aren't you injecting a singleton instance?
Consider using constructor based injections, as @ragore09 said comments above.

Reply:
done